src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,19 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class LoginPage extends MainPage{
```

Comment:
a co to jest LoginPage? Czy jest jakaś strona do logowania osobna bo w sumie Index Page się logowaliśmy.